.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Canary Release
+
+on:
+  workflow_run:
+    workflows: ["Build and Test"]
+    branches: [main]
+    types: 
+    - completed
+
+jobs:
+  canary-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Generate Release Manifests
+      run: make generate-manifests IMAGE=envoyproxy/gateway-dev TAG=latest OUTPUT_DIR=release-artifacts
```

Comment:
tag here should match the tag name below

Reply:
This tag is the image tag, built by build-and-test workflow, we do not have canary tag, we just have the latest tag and a commit id tag.

Reply:
yeah im recommending they both be the same

Reply:
Let us call them both 'latest'? Tag name is latest too.

Reply:
I do not suggest building an image with tag called canary, people often use latest instead, if we want them to be same, let us call them both latest?

Reply:
yeah `latest` is a good start and it won't break anything if `Build and Test` fails :trollface:

Reply:
Thanks, updated.